assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,137 @@
 console.log('***** Music Collection *****')
+
+// Created a variable called collection and
+// the function addToCollection
+
+let collection = [];
+
+function addToCollection (title, artist, year) {
+    let album = {
+        title,
+        artist,
+        yearPublished: year,
+    };
+    collection.push(album)
+    return collection;
+}
+
+// let album1 = {
+//     title: 'Hopes And Fears',
+//     artist: 'Keane',
+//     yearPublished: '2004',
+//     tracks: ['Somewhere Only We Know', 'Everybody\'s Changing', 'This Is the Last Time']
+// }
+
+// let album2 = {
+//     title: 'A Thousand Suns',
+//     artist: 'Linkin Park',
+//     yearPublished: '2010',
+//     tracks: ['Burning In the Skies', 'Robot Boy', 'Waiting for the End']
+// }
+// let album3 = {
+//     title: 'How To Conquer A Land',
+//     artist: 'And The Golden Choir',
+//     yearPublished: '2017',
+//     tracks: {
+//         Name:'How to Conquer A Land',
+//         Duration: '3:44'
+//     }
+// }
+// let album4 = {
+//     title: 'Riot!',
+//     artist: 'Paramore',
+//     yearPublished: '2007',
+//     tracks: ['That\'s What You Get', 'Misery Business', 'crushcrushcrush']
+// }
+// let album5 = {
+//     title: 'Hot Fuss',
+//     artist: 'The Killers',
+//     yearPublished: '2004',
+//     tracks: ['Mr. Brightside', 'Smile Like You Mean It']
+// }
+// let album6 = {
+//     title: 'Minutes to Midnight',
+//     artist: 'Linkin Park',
+//     yearPublished: '2007',
+//      tracks: ['What I\'ve Done', 'Shadow of the Day', 'In Pieces']
+// }
+
+// Added each album to collection. Logging each one in connsole and then
+// logging the collection array.
+
+addToCollection("Hopes And Fears", "Keane", "2004");
+console.log(collection[0]);
```

Comment:
If `addToCollection` returned the new object as expected, you could have tested this like this:

```js
console.log(addToCollection("Hopes And Fears", "Keane", "2004"));
```